pkg/storage/tableland/common/sqlfactory.go

```diff
@@ -55,6 +55,23 @@ func (s *SQLFactory) SQLForInsertingLayers(table string, layers []local.Layer) (
 	return clean(sql), nil
 }
 
+// SQLForInsertingRigs retutns the SQL statement.
+func (s *SQLFactory) SQLForInsertingRigs(table string, rigs []local.Rig) (string, error) {
+	var vals [][]interface{}
+	for _, rig := range rigs {
+		vals = append(
+			vals,
+			goqu.Vals{rig.ID, rig.RendersCid},
+		)
+	}
+	ds := s.d.Insert(table).Cols("id", "renders_cid").Vals(vals...)
+	sql, _, err := ds.ToSQL()
+	if err != nil {
+		return "", fmt.Errorf("creating sql to insert rigs: %v", err)
+	}
+	return clean(sql), nil
+}
+
```

Comment:
Transform data stored in our local SQLite db into SQL insert statements to populate our new rigs table on Tableland.